iwyu.cc

```diff
@@ -3704,7 +3704,11 @@ class IwyuAstConsumer
     // though, because that will drag in every overload even if we're
     // only using one.  Instead, we keep track of the using decl and
     // mark it as touched when something actually uses it.
-    preprocessor_info().FileInfoFor(CurrentFileEntry())->AddUsingDecl(decl);
+    auto fileInfo = preprocessor_info().FileInfoFor(CurrentFileEntry());
```

Comment:
We typically don't use `auto` unless the type is already visible on the line.

Hacker-case for variables, so `fileInfo` -> `file_info`.